### PR TITLE
Fix a bunch more warnings from eslint

### PR DESCRIPTION
**Pull request by alexwlchan**
Follows https://github.com/wellcomecollection/wellcomecollection.org/pull/8324

Before:

```console
$ yarn run eslint --fix --max-warnings=0 .
✖ 190 problems (10 errors, 180 warnings)
```

After:

```console
$ yarn run eslint --fix --max-warnings=0 .
✖ 165 problems (9 errors, 156 warnings)
```

### CODE REVIEW

File: common/views/components/Message/Message.tsx [hunk 1]
```diff
@@ -20,5 +21,5 @@ type Props = {
   text: string;
 };
 
-const Message = ({ text }: Props) => <Wrapper>{text}</Wrapper>;
+const Message: FC<Props> = ({ text }: Props) => <Wrapper>{text}</Wrapper>;
```
Comment (davidpmccormick) [inline common/views/components/Message/Message.tsx:24]: You can remove the second mention of `Props` here, now

File: common/views/components/PrismicHtmlBlock/PrismicHtmlBlock.tsx [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 import * as prismicT from '@prismicio/types';
+import { FC } from 'react';
 
 type Props = {
   html: prismicT.RichTextField;
   htmlSerializer?: JSXFunctionSerializer;
 };
 
-const PrismicHtmlBlock = ({ html, htmlSerializer }: Props) => (
+const PrismicHtmlBlock: FC<Props> = ({ html, htmlSerializer }: Props) => (
```
Comment (davidpmccormick) [inline common/views/components/PrismicHtmlBlock/PrismicHtmlBlock.tsx:11]: Same

**Review by davidpmccormick — APPROVED** [review]
There are some surplus to requirement `Props`, but I don't think it makes any odds

**Comment by alexwlchan** [comment]
Might as well fix them while I'm here!

**Pull request merged**